### PR TITLE
Add test for URL unshortening service

### PR DESCRIPTION
**Pull request by singhpratyush**
### Short description
This test case uses shortened URLs to validate proper functioning of URL unshortening service in loklak.

The tests will be failing right now because the issue will be fixed in #1122.

I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only strictly only one commit per issue.

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

**Comment by singhpratyush** [comment]
@kavithaenair: The build is failing because of buggy unshortening service. I have fixed it in the PR I mentioned in the description.

File: test/org/loklak/harvester/RedirectUnshortenerTest.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        shortlinkMap.put("http://t.co/E3w7s2qdBT", "http://mostviralfeed.com/what-lady-gaga-actually-looks-like");
+        shortlinkMap.put("http://bit.ly/1h9gTTT", "http://www.ebay.com/sch/i.html?_from=R40&_nkw=Pontiac+Catalina+1963&_in_kw=1&_ex_kw=&_sacat=0&_okw=&_oexkw=&_adv=1&_udlo=&_udhi=&LH_BIN=1&LH_PayPal=1&LH_Time=1&_ftrt=901&_ftrv=1&_sabdlo=&_sabdhi=&_samilow=&_samihi=&_sadis=200&_fpos=&_fsct=&LH_SALE_CURRENCY=0&_fss=1&_saslop=1&_sasl=&_fsradio=LH_SellerWithStore%3D1&_sop=12&_dmd=1&_ipg=50&rmvSB=true");
+        shortlinkMap.put("http://ow.ly/JtOPA", "http://yacy.net/en/index.html");
+        shortlinkMap.put("http://bit.do/ZwrT", "http://yacy.net/en/index.html");
+        shortlinkMap.put("http://fb.me/4lcXZsyyO", "https://www.facebook.com/permalink.php?story_fbid=1550262581900846&id=1381813572079082");
+        shortlinkMap.put("http://wp.me/p4yQu6-za0", "https://www.oklahomafoodandmusic.com/?p=135160");
+
+        for (HashMap.Entry<String, String> entry: shortlinkMap.entrySet()) {
+            String unshortenedURL = RedirectUnshortener.unShorten(entry.getKey());
+            assertEquals(entry.getValue(), unshortenedURL);
```
Comment (daminisatya) [inline test/org/loklak/harvester/RedirectUnshortenerTest.java]: @singhpratyush this is the failure please check

image

Comment (vibhcool) [inline test/org/loklak/harvester/RedirectUnshortenerTest.java]: Since this is a url, not a string, you can use assertThat() instead of assertEquals

**Comment by singhpratyush** [comment]
#1183 being merged, tests pass now.

**Comment by singhpratyush** [comment]
@daminisatya: Please update your review.

**Pull request merged**